lib/scrabble.rb

```diff
@@ -15,4 +12,67 @@ def point_values
       "Y"=>4, "Z"=>10
     }
   end
+
+  def score_with_multipliers(word, letter_mult, tot_multiplier=1)
+      case word_evaulator(word)
+      when false
```

Comment:
You're checking for falseyness and truthiness, but we would prefer an explicit check.